### ISSUE TITLE
Google charts stopped loading

### ISSUE THREAD

**Issue by ashwani-ksprof** [google/google-visualization-issues#2864]
We are facing issue with google charts. They are not loading and giving below error for almost all the charts.

**Uncaught (in promise) TypeError: google.visualization.ColumnChart is not a constructor**

They were working till yesterday (12-October-2020) 3 PM, but after that all of a sudden they stopped working.
I checked on the Google charts developer page and there also this problem occurs.

Please help to resolve the issue ASAP.

**Comment by dlaliberte** [comment]
Nothing has changed that I am aware of.   The ColumnChart seems to be working fine on the [documentation page](https://developers.google.com/chart/interactive/docs/gallery/columnchart). 

If you can point us at your web page, maybe we can figure out what is happening.

**Comment by ashwani-ksprof** [comment]
Yes, I noticed that the issue is with my Chrome, when I tried with Edge it works perfectly.
I think this issue is related to the below message I am getting in my console.

Uncaught ReferenceError: gvjs_tK is not defined
    at jsapi_compiled_corechart_module.js:2

It start occurring all of a sudden.

**Comment by dlaliberte** [comment]
That kind of unintentional error indicates there is a bug in Google Charts.  If you could point us at your web page, then we would have a chance of debugging it, so we might be able to suggest a workaround.  It would help more if you could narrow down the cause with a simple example, perhaps using jsfiddle.
